ui/components/multichain/pages/send/components/domain-input-resolution-cell.tsx

```diff
@@ -120,24 +124,14 @@ export const DomainInputResolutionCell = ({
                 size={AvatarIconSize.Xs}
                 className="multichain-send-page__recipient__item__avatar"
                 backgroundColor={BackgroundColor.infoDefault}
+                color={IconColor.infoInverse}
                 borderColor={BorderColor.backgroundDefault}
                 borderWidth={2}
-                iconProps={{
-                  color: IconColor.infoInverse,
-                  style: { width: '12px', height: '12px' },
-                  name: IconName.Snaps,
-                }}
               />
             }
-            positionObj={{
-              bottom: '25%',
-              right: '10%',
-            }}
-            badgeContainerProps={{
-              className: 'multichain-send-page__recipient__item__badge',
-            }}
+            position={BadgeWrapperPosition.bottomRight}
           >
-            <Identicon address={address} diameter={32} />
+            <AvatarAccount address={address} />
```

Comment:
You can use `AvatarAccount` here instead of Identicon it provides sizes, defaults to `32px`